src/Service/ResponseProcessor/Rest/BarcodeServiceRestResponseProcessor.php

```diff
@@ -63,18 +63,12 @@ class BarcodeServiceRestResponseProcessor extends AbstractRestResponseProcessor
     public function processGenerateBarcodeResponse(ResponseInterface $response): string
     {
         $this->validateResponse(response: $response);
-        $responseContent = $this->getResponseText(response: $response);
+        $body = json_decode(json: static::getResponseText(response: $response));
```

Comment:
Maybe these statements can be combined later, because they are always called together.

Reply:
Sounds like a good idea!